pkg/controller/replicaset/replica_set.go

```diff
@@ -382,14 +374,7 @@ func (rsc *ReplicaSetController) deletePod(obj interface{}) {
 			glog.Errorf("Couldn't get key for ReplicaSet %#v: %v", rs, err)
 			return
 		}
-		// This method only manages expectations for the case where a pod is
-		// deleted without a grace period.
-		if pod.DeletionTimestamp == nil {
-			glog.V(4).Infof("Received new delete for rs %v, pod %v", rsKey, pod.Name)
-			rsc.expectations.DeletionObserved(rsKey)
-		} else {
-			glog.V(4).Infof("Received delete for rs %v pod %v with non nil deletion timestamp %+v. Not counting it as a new deletion.", rsKey, pod.Name, pod.DeletionTimestamp)
-		}
+		rsc.expectations.DeletionObserved(rsKey, controller.PodKey(pod))
```

Comment:
In the `Pod %s deleted` log message above, it would be useful to also print the namespace and deletion timestamp.